tjreports/administrator/views/tjreport/view.html.php

```diff
@@ -96,14 +104,16 @@ protected function addToolBar()
 	 */
 	protected function addDocumentHeaderData()
 	{
-		JHtml::_('jquery.framework');
-		JText::script('COM_TJREPORTS_FORM_DEFAULT_OPTION');
-		JText::script('COM_TJREPORTS_INVALID_JSON_VALUE');
-		$document = JFactory::getDocument();
-		$document->addScript(JURI::root() . '/components/com_tjreports/assets/js/tjrContentService.js');
-		$document->addScript(JURI::root() . '/components/com_tjreports/assets/js/tjrContentUI.js');
-		$document->addStylesheet(JURI::root() . '/components/com_tjreports/assets/css/tjreports.css');
-		$document->addScriptDeclaration('tjrContentUI.base_url = "' . Juri::base() . '"');
-		$document->addScriptDeclaration('tjrContentUI.root_url = "' . Juri::root() . '"');
+		HTMLHelper::_('jquery.framework');
+		Text::script('COM_TJREPORTS_FORM_DEFAULT_OPTION');
+		Text::script('COM_TJREPORTS_INVALID_JSON_VALUE');
+
+		$document = Factory::getDocument();
+		$document->addScript(Uri::root() . '/components/com_tjreports/assets/js/tjrContentService.min.js');
+		$document->addScript(Uri::root() . '/components/com_tjreports/assets/js/tjrContentUI.min.js');
+		$document->addStylesheet(Uri::root() . '/components/com_tjreports/assets/css/tjreports.css');
+
+		$document->addScriptDeclaration('tjrContentUI.base_url = "' . Uri::base() . '"');
```

Comment:
@pranotiTechjoomla Replace this with Joomla script options

Reply:
done